### PR TITLE
feat: add StatusTag

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/4794600988

### CODE REVIEW

File: src/styles/tokens/colors.scss [hunk 1]
```diff
@@ -28,6 +28,7 @@
   --sb-disabled-background-color: #ecedf5;
   --sb-primary-background-hover-color: #dcdfec;
   --sb-dark-background-color: #f6f7fb;
+  --sb-brand-color: #5034ff;
```
Comment (SergeyRoyt) [inline src/styles/tokens/colors.scss:31]: Nice!
Let's also create a task to replace this hard-coded value to this token in vibe docs? We can put it for the hackathon :)

**Review by SergeyRoyt — APPROVED** [review]
Great!

**Comment by SergeyRoyt** [comment]
Only haven't understand why you are not merging it to master, but to other branch

File: src/styles/tokens/colors.scss (hunk 1 above)
Comment (talkor) [reply src/styles/tokens/colors.scss:31]: Good idea :)

**Comment by talkor** [comment]
> Only haven't understand why you are not merging it to master, but to other branch Also let's put a link to the corresponding monday task into description 🙏

I thought I'll be using some changes from the other branch but eventually it didn't matter, I can rebase

**Pull request closed without merging**